### PR TITLE
rename and upgrade postgres and redis values

### PR DESCRIPTION
**Pull request by alvarorm22**
**Description of the change**

Rename postgres and redis param name values
Upgrade postgres and redis charts templates

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

### CODE REVIEW

File: chart/values.yaml [hunk 1]
```diff
@@ -3429,19 +3429,22 @@ workspaceMigrations:
...
+## @param internalRedis.auth.password Redis&trade; password
+## @param internalRedis.auth.existingSecret Name of existing secret object containing the password
+## @param internalRedis.architecture Cluster settings
+## @param internalRedis.master.persistence.enabled Enable master persistent volumes
+## @param internalRedis.replica.persistence.enabled Enable replica persistent volumes
 ##
-redis:
+internalRedis:
   enabled: true
+  ## @param nameOverride String to partially override common.names.fullname template (will maintain the release name)
```
Comment (ManuelLR) [inline chart/values.yaml:3445]:
```suggestion
  ## @param internalRedis.nameOverride String to partially override common.names.fullname template (will maintain the release name)
```

File: chart/values.yaml [hunk 2]
```diff
@@ -3478,19 +3481,22 @@ externalRedis:
...
+## @param internalPostgresql.auth.existingSecret Name of an existing secret containing the PostgreSQL password ('postgresql-password' key)
 ##
-postgresql:
+internalPostgresql:
   enabled: true
   ## @param postgresql.image.tag Tag of the PostgreSQL image
   ##
   image:
     tag: "13.5.0-debian-10-r84"
+  ## @param nameOverride String to partially override common.names.fullname template (will maintain the release name)
```
Comment (ManuelLR) [inline chart/values.yaml:3497]:
```suggestion
  ## @param internalPostgresql.nameOverride String to partially override common.names.fullname template (will maintain the release name)
```

File: chart/templates/cdn-invalidator-sub/deployment.yaml [hunk 3]
```diff
@@ -175,4 +175,4 @@ spec:
         {{- if .Values.cdnInvalidatorSub.extraVolumes }}
         {{- include "common.tplvalues.render" (dict "value" .Values.cdnInvalidatorSub.extraVolumes "context" $) | nindent 8 }}
         {{- end }}
-{{- end }}
+{{- end }}
```
Comment (ManuelLR) [inline chart/templates/cdn-invalidator-sub/deployment.yaml:178]: :sob:

**Review by ManuelLR — COMMENTED** [review]
Minor comments in a quick :eyes:

File: chart/templates/_helpers.tpl [hunk 4]
```diff
@@ -655,36 +655,36 @@ Get the Postgresql credentials secret.
...
 {{- end -}}
 
 {{/*
 Add environment variables to configure database values
 */}}
 {{- define "carto.postgresql.secret.key" -}}
-{{- if .Values.postgresql.enabled -}}
-    {{- printf "%s" "password" -}}
+{{- if .Values.internalPostgresql.enabled -}}
+    {{- printf "%s" "postgres-password" -}}
```
Comment (ManuelLR) [inline chart/templates/_helpers.tpl:687]: Is that ok?

Comment (alvarorm22) [reply chart/templates/_helpers.tpl]: No, fixed

**Review by ManuelLR — APPROVED** [review]
LGTM

**Comment by alvarorm22** [comment]
https://app.shortcut.com/cartoteam/story/204140/helm-chart-configuration-and-customer-package

**Pull request merged**